src/pages/sym.mdx

```diff
@@ -0,0 +1,133 @@
+Extra tokens are currently not specified, but may be so in the future; so, implementations should ignore them, and may produce a warning in doing so.
+
+#### Symbols
+
+A symbol is uniquely defined by the combination of its location and its name; symbol names are case-sensitive.
+If the same symbol (identical name *and* identical location) is defined twice, one of the two definitions must be silently ignored.
+
+If two distinct symbols are defined with the same name (but different locations), neither definition shall be ignored, and this should not be an error.
+A warning may be produced, though not necessarily when loading the sym file: for example, "displaying names instead of memory addresses" accomodates this case just fine, however a "go to symbol" feature wouldn't, and a warning could be produced when the user tries to go to an ambiguous target.
+
+##### Location
+
+The location shall be of the form `<bank>:<address>`, both `<bank>` and `<address>` being case-insensitive hexadecimal numbers, possibly zero-padded, but without any prefix (so `$02:4000` is not a valid location, for example).
+An implementation may also allow the bank to be the case-insensitive keyword `BOOT` to specify symbols in the loaded boot ROM.
+If the bank is omitted, then the colon must be as well; the results of such lines are implementation-defined.
+
+The address must be in 16-bit range (0 to $FFFF inclusive), the bank must be in 32-bit range (0 to $FFFFFFFF inclusive); implementations may abandon processing the line if either number is larger than these limits.
+Implementations may accept bank numbers higher than $FFFFFFFF.
+Nonsensical locations (e.g. `2:9800`) may produce a warning.
+
+Two locations are considered identical if they have the same bank and the same address.
+Two distinct symbols can have identical locations; in particular, <LatestStableLink to="rgbasm.5#Unions">unions</LatestStableLink> facilitate this.
+
+Locations are strictly ordered by memory region (ROM0, ROMX, VRAM, SRAM, WRAM0, WRAMX, Echo RAM, OAM, I/O, HRAM, IE), then by bank, then by address.
+Entries may or may not be sorted in the sym file; for example, RGBLINK before v0.4.0 output symbols in the same order that it read them from its input files.
+Some sym files are piped through [`sort(1)`](https://man.openbsd.org/sort.1), which often produces files sorted by bank then address, but not by region.
+
+Sym files do not carry information about the memory region a symbol belongs to, but that can be inferred from the address most of the time.
+This detection is sometimes ambiguous: for example, `0:A000` could be one past the end of VRAM, or at the beginning of SRAM.
+The following heuristics are suggested:
+- If the symbol's bank is in range of exactly one of the candidate regions, assign it to that one. (*There should be no ambiguity, e.g. `2:A000`.*)
```

Comment:
How is `2:A000` ambiguous? There's no VRAM2, so it must be the start of SRAM bank 2.

Reply:
It is ambiguous if you only consider the address, which a naïve implementation might. As the line says, those cases actually have no ambiguity.

Reply:
It says "There should be no ambiguity, e.g. `2:A000`." I can only read this as "There should be no ambiguity, such as this ambiguous example, `2:A000`." How about (removing the italics too): `(There should be no ambiguity, `1:8000`, which can be in ROMX or VRAM.)`